iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/pipe/event/EnrichedEvent.java

```diff
@@ -148,6 +152,9 @@ public boolean decreaseReferenceCount(String holderMessage, boolean shouldReport
             Thread.currentThread().getStackTrace());
       }
     }
+    if (!isSuccessful) {
```

Comment:
It seems that the flag is meaningless and the "0" reference count will still go "<0" and succeed.

Reply:
The `isSuccessful` flag is used to determine whether internal increases or decreases were successful. In the case where the reference count is less than zero, we still consider the decrease to be successful.